chainercv/links/model/vgg/vgg16.py

```diff
@@ -0,0 +1,156 @@
+    The value of :obj:`VGG16.feature_names` selects the features that are going
+    to be collected by :meth:`__call__`.
+    :obj:`self.all_feature_names` is the list of the names of features
+    that can be collected.
+
+    Examples:
+
+        >>> model = VGG16()
+        # By default, VGG16.__call__ returns a probability score.
+        >>> prob = model(imgs)
+
+        >>> model.feature_names = 'conv5_3'
+        # This is feature conv5_3.
+        >>> feat5_3 = model(imgs)
+
+        >>> model.feature_names = ['conv5_3', 'fc6']
+        >>> # These are features conv5_3 and fc6.
+        >>> feat5_3, feat6 = model(imgs)
+
+    .. seealso::
+        :class:`chainercv.links.model.SequentialFeatureExtractor`
+
+    When :obj:`pretrained_model` is the path of a pre-trained chainer model
+    serialized as a :obj:`.npz` file in the constructor, this chain model
+    automatically initializes all the parameters with it.
+    When a string in the prespecified set is provided, a pretrained model is
+    loaded from weights distributed on the Internet.
+    The list of pretrained models supported are as follows:
+
+    * :obj:`imagenet`: Loads weights trained with ImageNet and distributed \
+        at `Model Zoo \
+        <https://github.com/BVLC/caffe/wiki/Model-Zoo>`_.
+
+    Args:
+        pretrained_model (str): The destination of the pre-trained
+            chainer model serialized as a :obj:`.npz` file.
+            If this is one of the strings described
+            above, it automatically loads weights stored under a directory
+            :obj:`$CHAINER_DATASET_ROOT/pfnet/chainercv/models/`,
+            where :obj:`$CHAINER_DATASET_ROOT` is set as
+            :obj:`$HOME/.chainer/dataset` unless you specify another value
+            by modifying the environment variable.
+        n_class (int): The number of classes.
+        mean (numpy.ndarray): A mean value. If :obj:`None` and
+            a supported pretrained model is used,
+            the mean value used to train the pretrained model will be used.
+        initialW (callable): Initializer for the weights.
+        initial_bias (callable): Initializer for the biases.
+
+    """
+
+    _models = {
+        'imagenet': {
+            'n_class': 1000,
+            'url': 'https://github.com/yuyu2172/share-weights/releases/'
+            'download/0.0.4/vgg16_imagenet_convert_2017_07_18.npz',
+            'mean': _imagenet_mean
+        }
+    }
+
+    def __init__(self,
+                 pretrained_model=None, n_class=None, mean=None,
+                 initialW=None, initial_bias=None):
+        if n_class is None:
+            if pretrained_model in self._models:
+                n_class = self._models[pretrained_model]['n_class']
+            else:
+                n_class = 1000
+
+        if mean is None:
+            if pretrained_model in self._models:
+                mean = self._models[pretrained_model]['mean']
+            else:
+                mean = _imagenet_mean
```

Comment:
How about adding a note about these behaviours? The default values of `n_class` and `mean`.